components/button/Button.module.css

```diff
@@ -44,3 +44,17 @@
 
   padding: 0.05em 0.22em;
 }
+
+.joinBtn {
+  --bottom-border-length: 62%;
+
+  font-size: 2.2em;
+  padding: 0.25em 0.22em;
+}
+
+.joinBackground {
+  background-color: var(--primary-color);
+  padding: 0.05em 0.23em;
+  box-shadow: 0 3px 10px #000000c7;
```

Comment:
Maybe also use a variable for the shadow color if you plan to use it in more components. Then you would only have to modify it in one place if you change your mind.